src/components/Map/index.tsx

```diff
@@ -1,15 +1,14 @@
 import { VStack, Box } from '@chakra-ui/react';
 import { useEffect, useRef, useState } from 'react';
 import MapButton from './MapButton';
+import { DEFAULT_LOCATION } from '@/utils/constance';
```

Comment:
```suggestion
import { INIT_LOCATION } from '@/utils/constance';
```

올리시기 전에 빌드 확인 해주셨나요?! 파편화되어있네요